src/node_platform.h

```diff
@@ -66,6 +67,7 @@ class PerIsolatePlatformData :
   int unref();
 
   // Returns true iff work was dispatched or executed.
+  // New tasks that are posted during flushing of the queue are not run.
```

Comment:
Tiny nit: Maybe be more specific here and say that they are not run as part of this iteration, rather than implying that they aren’t run at all?

Reply:
Thanks. Done.